index.js

```diff
@@ -165,6 +166,11 @@ module.exports = function (cmd, args, opts) {
 
 			err.stdout = stdout;
 			err.stderr = stderr;
+
+			if (!parsed.opts.reject) {
+				return err;
```

Comment:
Hmm, yeah, some props to differentiate this from a succeeding result would be nice. Dunno about changing the `code` though. You might want the original one.